src/pydot/dot_parser.py

```diff
@@ -383,7 +385,7 @@ def push_node_stmt(s: str, loc: int, toks: ParseResults) -> "pydot.core.Node":
     return n
 
 
-class HTML(Token):
+ class HTML(Token):
```

Comment:
No idea what happened here.
```suggestion
class HTML(Token):
```